interface/src/state/mod.rs

```diff
@@ -10,10 +10,12 @@ pub type COption<T> = ([u8; 4], T);
 
 /// Marker trait for types that can be cast from a raw pointer.
 ///
+/// # Safety
+///
 /// It is up to the type implementing this trait to guarantee that the cast is
 /// safe, i.e., the fields of the type are well aligned and there are no padding
 /// bytes.
```

Comment:
To go with this safety concern, what do you think about specifying that `bytes` must be on a well-aligned address for `T` for all the `load` functions?

Reply:
Good point, I will add a note on `load` in a separate PR.